keps/sig-apps/961-maxunavailable-for-statefulset/README.md

```diff
@@ -373,31 +902,68 @@ The struct in question is IntOrString.
 No
 
 ###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
-The controller-manager will see very negligible and almost un-notoceable increase in cpu usage.
+The controller-manager will see very negligible and almost un-noticeable increase in cpu usage.
+
+###### Can enabling / using this feature result in resource exhaustion of some node resources (PIDs, sockets, inodes, etc.)?
+
+No.
 
 ### Troubleshooting
 
 ###### How does this feature react if the API server and/or etcd is unavailable?
-The RollingUpdate will fail or will not be able to proceed if etcd or apiserver is unavailable and
+The RollingUpdate will fail or will not be able to proceed if etcd or API server is unavailable and
 hence this feature will also be not be able to be used.
 
 ###### What are other known failure modes?
-NA
+
+<!--
+For each of them, fill in the following information by copying the below template:
+  - [Failure mode brief description]
+    - Detection: How can it be detected via metrics? Stated another way:
+      how can an operator troubleshoot without logging into a master or worker node?
+    - Mitigations: What can be done to stop the bleeding, especially for already
+      running user workloads?
+    - Diagnostics: What are the useful log messages and their required logging
+      levels that could help debug the issue?
+      Not required until feature graduated to beta.
+    - Testing: Are there any tests for failure mode? If not, describe why.
+-->
+
+
+- Incorrect Handling of minReadySeconds During StatefulSet Updates with Parallel Pod Management
+  - Detection: 
+    - Monitor the status.availableReplicas of the StatefulSet during rolling updates. A significant drop below the expected number of available replicas, considering the maxUnavailable setting, could indicate the issue.
+    - Review StatefulSet events or controller logs for rapid succession of pod updates without adherence to minReadySeconds, which could confirm that the delay is not being respected.
+  - Mitigations:
+    - Temporarily adjust the podManagementPolicy to OrderedReady as a workaround to ensure minReadySeconds is respected during updates, though this may slow down the rollout process.
+    - If feasible, review and adjust minReadySeconds and maxUnavailable settings to mitigate the impact until a fix is applied.
+    - Upgrade your cluster to 1.30, where this feature is promoted to beta, and the issue is fixed.
+  - Diagnostics:
+    - Enable detailed logging for the StatefulSet controller to capture the sequence and timing of pod updates during a rollout.
```

Comment:
we can mention that first option can be just looking at the events. If that is not suffient logging should be used.

Reply:
✅